cicecore/cicedyn/general/ice_flux.F90

```diff
@@ -609,6 +609,11 @@ subroutine alloc_flux
          stat=ierr)
       if (ierr/=0) call abort_ice('(alloc_flux): Out of memory')
 
+         swuvrdr(:,:,:) = c0
+         swuvrdf(:,:,:) = c0
+         swpardr(:,:,:) = c0
+         swpardf(:,:,:) = c0
```

Comment:
Looks good, please shift indentation to the left.

Reply:
Done.